.travis.yml

```diff
@@ -35,7 +35,9 @@ jobs:
         cp -r documentation/docs documentation/docs-tmp
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
-    
+    - name: Consumer Tests for s4sdk pipeline
```

Comment:
are the consumer tests executed for every PR voting?
If so, this will dramatically reduce feedback time especially when adding additional scenarios in the future.

Another option could be to
* only trigger consumer tests on request (e.g. github tag, ...)
* trigger consumer tests only in master pipeline

Reply:
We'd like to know if something breaks before it goes to master, so I think we should have it for each PR.